03-Custom-Actions-In-Python/04-Custom-Updater.ipynb

```diff
@@ -0,0 +1,386 @@
+{
```

Comment:
<p>I was copying each code block into a single python file to run and it took me a sec to realize this wasn't related to updaters. This is helpful to see but it might be out of scope for this tutorial.  Maybe link to the tutorial on variants instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>We don't currently have one, and I don't know if one would be in the works.</p>

Reply:
The introduction tutorial briefly explains variants: https://hoomd-blue.readthedocs.io/en/latest/tutorial/01-Introducing-Molecular-Dynamics/03-Compressing-the-System.html#Variants

If a link is warranted, use a relative link and sphinx/nbconvert should handle it.